### PR TITLE
more client mappings!

### LINKED ISSUE #453: `SignedArgument` is an incorrect name
the class `C_wdcbkyfy` is named SignedArgument, when in reality it is a record containing a list of arguments:
screenshot showing the class declaration, which shows it stores a list of arguments

### PR DESCRIPTION
**Pull request by eristhea**
- completes fonts, finally!
- completes GuiGraphics, again!
- more general client mappings

builds off of #469, avoid reviewing until that's merged.
fixes #453 
falls victim to #470

### CODE REVIEW

File: mappings/net/minecraft/client/font/CodepointMap.mapping [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	METHOD m_eikegste forEach (Lnet/minecraft/unmapped/C_srmdcrvq$C_xbamquui;)V
+		ARG 1 glyphConsumer
+	METHOD m_higtkvia clear ()V
+	METHOD m_pepaswby put (ILjava/lang/Object;)Ljava/lang/Object;
+		ARG 1 codePoint
+		ARG 2 glyph
+	METHOD m_prbxgcyk keySet ()Lit/unimi/dsi/fastutil/ints/IntSet;
+	METHOD m_qaadkitj get (I)Ljava/lang/Object;
+		ARG 1 codePoint
+	METHOD m_qntqpivy computerIfAbsent (ILjava/util/function/IntFunction;)Ljava/lang/Object;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/font/CodepointMap.mapping]:
```suggestion
	METHOD m_qntqpivy computeIfAbsent (ILjava/util/function/IntFunction;)Ljava/lang/Object;
```

File: mappings/net/minecraft/client/font/CodepointMap.mapping [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		ARG 1 codePoint
+		ARG 2 ifAbsent
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/font/CodepointMap.mapping]:
```suggestion
		ARG 2 mappingFunction
```
to be inline with Map#computeIfAbsent: https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#computeIfAbsent-K-java.util.function.Function-

Also to communicate that the value returned by the function will be put into the map; `ifAbsent` sounds to me like it's a runnable that runs if absent.

File: mappings/net/minecraft/client/network/ClientPlayerEntity.mapping [hunk 3]
```diff
@@ -93,6 +93,8 @@ CLASS net/minecraft/unmapped/C_jlopmfei net/minecraft/client/network/ClientPlaye
 	METHOD m_sliqpvmd canSprint ()Z
 	METHOD m_spcgxwbg setClientPermissionLevel (I)V
 		ARG 1 clientPermissionLevel
+	METHOD m_syghdsvg onGameModeChanged (Lnet/minecraft/unmapped/C_lghcpyvl;)V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/network/ClientPlayerEntity.mapping:93]: Not so much an issue with this in particular, but the tense of `onOccurrence` event-handler-like methods seems inconsistent afaict.

Why is this past tense but `ClientPlayerEntity#onPickupSlotClick` is not?

This seems like something that should be in CONVENTIONS.md, either clarifying that they should all be one way, or clarifying when to use one way or the other.

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/client/network/ClientPlayerEntity.mapping (hunk 3 above)
Comment (eristhea) [reply mappings/net/minecraft/client/network/ClientPlayerEntity.mapping:93]: I'm definitely in favor of past tense here, I agree it should be clarified

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by EnnuiL — APPROVED** [review]

File: mappings/net/minecraft/client/font/BitmapFont.mapping [hunk 4]
```diff
@@ -2,6 +2,29 @@ CLASS net/minecraft/unmapped/C_tfzxpuio net/minecraft/client/font/BitmapFont
 	FIELD f_mrmzkoex image Lnet/minecraft/unmapped/C_ayikuhxa;
 	FIELD f_pjauldcv LOGGER Lorg/slf4j/Logger;
 	FIELD f_syldtzsw glyphs Lnet/minecraft/unmapped/C_srmdcrvq;
+	CLASS C_qoeqnynz Definition1
```
Comment (IotaBread) [inline mappings/net/minecraft/client/font/BitmapFont.mapping]: Is the `1` intentional? Looks like a typo

Comment (eristhea) [reply mappings/net/minecraft/client/font/BitmapFont.mapping]: oh yeah it is. [this enigma bug](https://github.com/QuiltMC/enigma/commit/92dab41ba0b23b8e23b9332d68734a281fdef7c4) meant I had to throw the `1` on there so it wasn't a conflict. just forgot to clean that one up :)

**Pull request merged**